jsplot/src/main/java/tech/tablesaw/plotly/traces/BarTrace.java

```diff
@@ -29,11 +32,25 @@ private BarTrace(BarBuilder builder) {
   }
 
   public static BarBuilder builder(Object[] x, double[] y) {
-    return new BarBuilder(x, y);
```

Comment:
I'm at a loss to say why we (I, probably) added so many methods that take an array of objects (or doubles or whatever) as arguments for plotting.  Maybe it was because I was envisioning a close integration with Smile which dealt with values in that form. 

At this point I would be inclined to deprecate those methods. The arguments to plot methods in Tablesaw should be tables and column names, or columns. 

Since you did the work already, i'll review it.

Reply:
You have 3 nearly identical methods where you have one Object[] and a double[]. Can you extract the common code into another method?